test/Birds.Unit.Tests/Usings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
what mean this?

Reply:
https://learn.microsoft.com/es-es/dotnet/core/project-sdk/overview#implicit-using-directives